src/content/deployment/flavors.md

```diff
@@ -144,26 +144,31 @@ the following steps:
     with developer options enabled.
 
 1.  In the console, navigate to the `flavors_example`
-    directory and enter the following command:
+    directory and use the appropriate command.
 
-    ```console title="console"
-    $ flutter (run | build) --flavor <flavor_name>
-    ```
-
-    * `(run |  build)`: Replace this with one of the
-      following:
-      * `run`: Run the app in debug mode.
-      * `build`: Run the app in production mode.
-
-    * `<flavor_name>`: Replace this with the name of your
-      Android product flavor (for example, `staging` or
-      `production`).
+    - **To run the app in debug mode on a specific flavor:**
+      ```console title="console"
+      $ flutter run --flavor <flavor_name>
+      ```
+      Example:
+      ```console title="console"
+      $ flutter run --flavor staging
+      ```
```

Comment:
I think we should try to make the changes as minimal as possible to fix the issue, something like this would work:

1.  In the console, navigate to the `flavors_example`
    directory and enter the following command:

    ```console title="console"
    $ flutter (run | build <subcomand>) --flavor <flavor_name>
    ```

    * `(run |  build <subcommand>)`: Replace this with one of the
      following:
      * `run`: Run the app in debug mode.
      * `build`: Build either an apk or an appbundle.
        * `<subcommand>`: Either `apk` or `appbundle`.